app/controllers/admin/api/backend_apis/services_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class Admin::Api::BackendApis::ServicesController < Admin::Api::BackendApis::BaseController
+  wrap_parameters Service, include: Service.attribute_names | %w[state_event]
+  representer Service
+
+  # TODO: ActiveDocs
+  # TODO: Refactor
+  # TODO: should it all be called product instead of service?
+  # TODO: paginate
+
+  def index
+    services = backend_api.services.accessible
+    respond_with(services)
+  end
+
+  def create
+    service = current_account.services.build
+    create_service = ServiceCreator.new(service: service, backend_api: backend_api)
+    create_service.call(service_params)
+    service.reload if service.persisted? # It has been touched # TODO: Add test for this
```

Comment:
I think this is not needed

Reply:
I think the same but i am investigating so far: https://github.com/3scale/porta/pull/1214/commits/1cbef5e33fe93f0f32f3389b525a3e1db355025f
the difference is the `updated_at`, so i think the key is to change the way we create to save at the end only, but we'll see...

Reply:
Anyway, let's keep the conversation in the other PR plz, this one is only for QE to be able do work in the meantime